cmd/version_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd_test
+
+import (
+	"testing"
+
+	"github.com/appsody/appsody/cmd/cmdtest"
+)
+
+func TestVersion(t *testing.T) {
+
+	args := []string{"version"}
+	_, err := cmdtest.RunAppsodyCmd(args, ".", t)
```

Comment:
This will need to change, since the sandboxing PR #788 will remove `RunAppsodyCmd`.

Reply:
Change to `RunAppsody(sandbox)`